docs/reference/query-dsl/mlt-query.asciidoc

```diff
@@ -116,18 +116,18 @@ curl -s -XPUT 'http://localhost:9200/imdb/' -d '{
     "movies": {
       "properties": {
         "title": {
-          "type": "string",
+          "type": "text",
           "term_vector": "yes"
          },
          "description": {
-          "type": "string"
+          "type": "text"
         },
         "tags": {
-          "type": "string",
+          "type": "text",
           "fields" : {
             "raw": {
-              "type" : "string",
-              "index" : "not_analyzed",
+              "type" : "text",
```

Comment:
this example needs to use text with a keyword analyzer since keyword fields do not support term vectors